readthedocs/storage/s3_storage.py

```diff
@@ -80,6 +69,28 @@ def __init__(self, *args, **kwargs):
         self.querystring_auth = False
 
 
+class S3StaticStorage(
+    S3StaticStorageMixin, OverrideHostnameMixin, S3ManifestStaticStorage, S3Boto3Storage
+):  # pylint: disable=too-many-ancestors
+
+    """
+    An AWS S3 Storage backend for static media.
+
+    * Uses Django's ManifestFilesMixin to have unique file paths (eg. core.a6f5e2c.css)
+    """
+
+
+class RTDS3StaticStorage(
+    S3StaticStorageMixin, OverrideHostnameMixin, S3Boto3Storage
+):  # pylint: disable=too-many-ancestors
+
+    """
+    Storage backend for static files used outside Django's static files.
+
+    This is the same as S3StaticStorage, but without inheriting from S3ManifestStaticStorage.
```

Comment:
I don't fully understand why we need this class. Is it because we're accessing hashed files directly? 

```suggestion
    This is the same as S3StaticStorage, but without inheriting from S3ManifestStaticStorage.
    We need this because..
```

Reply:
yes